Sasquatch/SasquatchSwift/AppCenterDelegateSwift.swift

```diff
@@ -60,6 +60,10 @@ class AppCenterDelegateSwift: AppCenterDelegate {
     MSAppCenter.startFromLibrary(withServices: [MSAnalytics.self])
   }
 
+  func setUserId(_ usedId: String) {
+    // TODO
```

Comment:
TODO with descriptions or you can write the actual code and comment it out. If someone else address this TODO, we need to know what to do here.